capstone/pom.xml

```diff
@@ -108,6 +108,17 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>com.google.appengine</groupId>
+      <artifactId>appengine-remote-api</artifactId>
+      <version>1.9.64</version>
+    </dependency>
+
+    <dependency>
```

Comment:
Should we perhaps replace the dependency on line 59 with this one just so that there's only one GAE SDK dependency?

Reply:
Yes, fixed!